test/python/compiler/test_transpiler.py

```diff
@@ -696,6 +699,37 @@ def test_measure_doesnt_unroll_ms(self, optimization_level):
         self.assertEqual(qc, out)
 
 
+class StreamHandlerRaiseException(StreamHandler):
```

Comment:
While this class is only 3 lines I feel like we should move it some where common to avoid having a copy in every place we're asserting log output.